File: Cypher.cs
public class Cypher {
	public static char[] p = {'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i',
        'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v',
        'w', 'x', 'y', 'z'};
    public static char[] ch = {'Q', 'W', 'E', 'R', 'T', 'Y', 'U', 'I', 'O',
        'P', 'A', 'S', 'D', 'F', 'G', 'H', 'J', 'K', 'L', 'Z', 'X', 'C',
        'V', 'B', 'N', 'M'};
	public static Regex regex = new Regex("[-+.^@_:,]");
	
	public static string doEncryption(string s) {
        //try to remove some special characters from the string as this cypher is not equipped for symbols
        s = regex.Replace(s, "");
        char[] c = new char[s.Length];
        for (int i = 0; i < s.Length; i++) {
            for (int j = 0; j < 26; j++) {
                if (p[j] == char.ToLower(s.ElementAt(i))) {
                    c[i] = ch[j];
                    break;
                }
            }
        }
        return (new string(c));
    }
	
	public static string doDecryption(string s) {
        char[] p1 = new char[s.Length];
        for (int i = 0; i < s.Length; i++) {
            for (int j = 0; j < 26; j++) {
                if (ch[j] == s.ElementAt(i)) {
                    p1[i] = p[j];
                    break;
                }
            }
        }
        return (new string(p1));
    }
}

File: Cypher.vb

Public Class Cypher
	Public Shared p As Char() = {"a"C, "b"C, "c"C, "d"C, "e"C, "f"C, _
		"g"C, "h"C, "i"C, "j"C, "k"C, "l"C, _
		"m"C, "n"C, "o"C, "p"C, "q"C, "r"C, _
		"s"C, "t"C, "u"C, "v"C, "w"C, "x"C, _
		"y"C, "z"C}
	Public Shared ch As Char() = {"Q"C, "W"C, "E"C, "R"C, "T"C, "Y"C, _
		"U"C, "I"C, "O"C, "P"C, "A"C, "S"C, _
		"D"C, "F"C, "G"C, "H"C, "J"C, "K"C, _
		"L"C, "Z"C, "X"C, "C"C, "V"C, "B"C, _
		"N"C, "M"C}
	Public Shared regex As New Regex("[-+.^@_:,]")

	Public Shared Function doEncryption(s As String) As String
		'try to remove some special characters from the string as this cypher is not equipped for symbols
		s = regex.Replace(s, "")
		Dim c As Char() = New Char(s.Length - 1) {}
		For i As Integer = 0 To s.Length - 1
			For j As Integer = 0 To 25
				If p(j) = Char.ToLower(s.ElementAt(i)) Then
					c(i) = ch(j)
					Exit For
				End If
			Next
		Next
		Return (New String(c))
	End Function

	Public Shared Function doDecryption(s As String) As String
		Dim p1 As Char() = New Char(s.Length - 1) {}
		For i As Integer = 0 To s.Length - 1
			For j As Integer = 0 To 25
				If ch(j) = s.ElementAt(i) Then
					p1(i) = p(j)
					Exit For
				End If
			Next
		Next
		Return (New String(p1))
	End Function
End Class